hieradata/staging.yaml

```diff
@@ -564,3 +564,5 @@ users::pentest_machines:
 
 mongodb::s3backup::backup::s3_bucket: 'govuk-mongodb-backup-s3-staging'
 mongodb::s3backup::backup::s3_bucket_daily: 'govuk-mongodb-backup-s3-daily-staging'
+
+trade_tariff_uri: 'https://tariff-frontend-staging.cloudapps.digital/'
```

Comment:
I think namespacing in that way might imply there's a class called `plek_overrides` so perhaps it'd be clearer if it were called `plek_trade_tariff_uri`. I think it's fine either way though, personally.